packages/reputation-miner/ReputationMiner.js

```diff
@@ -1313,67 +1346,59 @@ class ReputationMiner {
     if (currentStateHash !== reputationRootHash) {
       console.log("WARNING: The supplied state failed to be recreated successfully. Are you sure it was saved?");
     }
-    await db.close();
   }
 
   async getAddressesWithReputation(reputationRootHash, colonyAddress, skillId) {
-    const db = await sqlite.open({filename: this.dbPath, driver: sqlite3.Database});
-    const res = await db.all(
-      `SELECT DISTINCT users.address as user_address, reputations.value as value
-       FROM reputations
-       INNER JOIN colonies ON colonies.rowid=reputations.colony_rowid
-       INNER JOIN users ON users.rowid=reputations.user_rowid
-       INNER JOIN reputation_states ON reputation_states.rowid=reputations.reputation_rowid
-       WHERE reputation_states.root_hash="${reputationRootHash}"
-       AND colonies.address="${colonyAddress.toLowerCase()}"
-       AND reputations.skill_id="${skillId}"
-       AND users.address!="0x0000000000000000000000000000000000000000"
-       ORDER BY reputations.value DESC`
-    );
-    await db.close();
+    const res = await this.queries.getAddressesWithReputation.all(reputationRootHash, colonyAddress.toLowerCase(), skillId);
     const addresses = res.map(x => x.user_address)
     return addresses;
   }
 
+  async getReputationsForAddress(reputationRootHash, colonyAddress, userAddress) {
+    const res = await this.queries.getReputationsForAddress.all(reputationRootHash, colonyAddress.toLowerCase(), userAddress.toLowerCase());
+    return res.map(function(x){ return {
```

Comment:
Newline before `return`

Reply:
There are... a lot of places where we don't do this? Is this a thing we do in javascript?

Reply:
Ok, never mind then.